packages/core/src/index.ts

```diff
@@ -1,30 +1,35 @@
 import { checkDomain, compileDomain, parseDomain } from "compiler/Domain";
 import { compileStyle } from "compiler/Style";
 import {
```

Comment:
Same spacing issue in this file again?

Reply:
I'd suggest doubling checking if you have [the Prettier extension](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) installed and enabled in VSCode.